sycl/test/lit.cfg.py

```diff
@@ -198,10 +198,8 @@ def getDeviceCount(device_type):
 aot_tools = ["opencl-aot", "ocloc", "aoc"]
 for aot_tool in aot_tools:
     if find_executable(aot_tool) is not None:
-        print("Found AOT device compiler " + aot_tool)
+        lit_config.note("Found AOT device compiler " + aot_tool)
         config.available_features.add(aot_tool)
-    else:
-        print("Could not find AOT device compiler " + aot_tool)
```

Comment:
Why did you remove this?

Reply:
I'm not sure that this one is very valuable: for example, we don't print "Not found available CPU/GPU/accelerator device". But if you want, I can return this message back

Reply:
I thought we print `getDeviceCount cpu:OpenCL runtime not found·` because it was in PR description. But in my environment I see `getDeviceCount gpu:PI_OPENCL` when I don't have GPU, this is pretty useless... I think we should print that we didn't find something too (including device runtimes and AOT tools), It could help to prevent hours of debugging nonexistent/missed problems.

Reply:
> I thought we print getDeviceCount cpu:OpenCL runtime not found· because it was in PR description. 

Oh, I clearly missed that one :)

I will return "aot device compiler not found" message and revise other messages to double-check that they are actually useful

Reply:
>  But in my environment I see getDeviceCount gpu:PI_OPENCL when I don't have GPU, this is pretty useless...

Should be fixed now

> . I think we should print that we didn't find something too (including device runtimes and AOT tools), It could help to prevent hours of debugging nonexistent/missed problems.

Done